From: "Matteo F. Vescovi" <mfv@debian.org>
Date: Sat, 29 Oct 2016 14:31:49 +0200
Subject: blender_thumbnailer

* Add blender.thumbnailer file
* Move thumbnailer script to an appropriate destination
---
 blender.thumbnailer           | 4 ++++
 source/creator/CMakeLists.txt | 2 +-
 2 files changed, 5 insertions(+), 1 deletion(-)
 create mode 100644 blender.thumbnailer

diff --git a/blender.thumbnailer b/blender.thumbnailer
new file mode 100644
index 000000000..05d4faeb9
--- /dev/null
+++ b/blender.thumbnailer
@@ -0,0 +1,4 @@
+[Thumbnailer Entry]
+TryExec=/usr/share/blender/scripts/blender-thumbnailer.py
+Exec=/usr/share/blender/scripts/blender-thumbnailer.py %i %o
+MimeType=application/x-blender;
diff --git a/source/creator/CMakeLists.txt b/source/creator/CMakeLists.txt
index f65688e13..2f5741753 100644
--- a/source/creator/CMakeLists.txt
+++ b/source/creator/CMakeLists.txt
@@ -486,7 +486,7 @@ if(UNIX AND NOT APPLE)
 		install(
 			PROGRAMS
 			${CMAKE_SOURCE_DIR}/release/bin/blender-thumbnailer.py
-			DESTINATION "."
+			DESTINATION ${CMAKE_INSTALL_PREFIX}/share/blender/scripts
 		)
 
 		set(BLENDER_TEXT_FILES_DESTINATION ".")
